src/pluggy/_hooks.py

```diff
@@ -541,10 +541,10 @@ def call_extra(
             hookimpl = HookImpl(None, "<temp>", method, opts)
             # Find last non-tryfirst nonwrapper method.
             i = len(hookimpls) - 1
-            while (
-                i >= 0
-                and hookimpls[i].tryfirst
-                and not (hookimpls[i].hookwrapper or hookimpls[i].wrapper)
+            while i >= 0 and (
```

Comment:
while just taking a look at line 531 it seems that #441 was incomplete as the specifically chosen arguments in the body of the method currently exclude the desired behavior as they intentional

cc @smarie

Reply:
@RonnyPfannschmidt not sure I get your point here. Do you mean that additional tests should be implemented in this PR to cover another situation ?

Reply:
No,what I meant is that based on the preceding code, the desired feature was intentionally unsupported and we need to do some design work to ensure its supported correctly instead of just enabling it